packages/cmf-cqrs/src/components/ACKDispatcher/ACKDispatcher.container.js

```diff
@@ -0,0 +1,73 @@
+import React, { PropTypes } from 'react';
+import { Map } from 'immutable';
+import { api, componentState } from 'react-cmf';
+
+import { deleteACK } from '../../actions/ack';
+
+export const DEFAULT_STATE = new Map({
+
+});
+/**
+ * {
+		type: 'ACK_ADD_CONTEXT',
+		requestId: '123',
+		data: { foo: 'bar' },
+		actionCreator: 'dataset:fetchAll'
+	}
+	{
+		type: 'ACK_RECEIVE_MESSAGE',
+		requestId: '123',
+	}
+
+ */
+class ACKDispatcher extends React.Component {
```

Comment:
a big fat comment should be put in this section, since i hope this component will disapear fast in favor of a a clean solution living in middleware realm.
this solution mean that many useless rerender may be trigger, and lot of other issue that could happen around race conditions.